charmap.asm

```diff
@@ -6,10 +6,10 @@
 	charmap "<PLAY_G>",  $14 ; "<PLAYER>くん" or "<PLAYER>ちゃん"; same as "<PLAYER>" in English
 	charmap "<MOBILE>",  $15
 	charmap "<CR>",      $16
-	charmap "¯",         $1f ; soft linebreak
+	charmap "<BSP>",     $1f ; breakable space (usually " ", but renders as line break on the Town Map)
 	charmap "<LF>",      $22
 	charmap "<POKE>",    $24 ; "<PO><KE>"
-	charmap "%",         $25 ; soft linebreak in landmark names
+	charmap "<ZWSP>",    $25 ; zero-width breakable space (usually not displayed, but renders as line break on the Town Map), unused
```

Comment:
I think `WBR` is more accurate and matches the HTML `<wbr>` tag.
```suggestion
	charmap "<WBR>",     $25 ; word-break opportunity (usually skipped, or "<LF>" on the Town Map)
```

Reply:
`<WBR>` is [equivalent to](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr) U+200B ZERO-WIDTH SPACE (which is commonly shortened to ZWSP), so they mean the same thing. However, given it is being rendered in angle-brackets, using the HTML element does make more sense (as people are more likely to be familiar with that name), and I think "word-break opportunity" is a more self-explanatory name. Happy to apply that change.